warehouse/query-core/src/main/java/datawave/webservice/modification/MutableMetadataHandler.java

```diff
@@ -921,16 +920,16 @@ columnVisibility, new Date(0), new Date(), StringUtils.join(auths, ','), expirat
             
             id = createResponse.getResult();
             BaseQueryResponse response = queryService.next(id);
-            if (response instanceof DefaultEventQueryResponse) {
-                DefaultEventQueryResponse eResponse = (DefaultEventQueryResponse) response;
+            if (response instanceof EventQueryResponseBase) {
+                EventQueryResponseBase eResponse = (EventQueryResponseBase) response;
                 if (eResponse.getEvents().size() > 1) {
                     throw new IllegalStateException("More than one event matched " + uuid + " (" + eResponse.getEvents().size() + " matched)");
                 }
                 if (eResponse.getEvents().isEmpty()) {
                     throw new IllegalStateException("No event matched " + uuid);
                 }
                 
-                e = (DefaultEvent) eResponse.getEvents().get(0);
+                e = eResponse.getEvents().get(0);
             }
```

Comment:
Should we throw some exception in the case where the response is not an instance of `EventQueryResponseBase`?

Reply:
but it appears that all exceptions are caught in this method denoting the event was not found (for whatever reason).  So I think I will leave this as is for now, but I will log something.